src/components/shared/Task.tsx

```diff
@@ -63,12 +64,12 @@ function Task(prop: {
         <h2 className="heading-1 task-name">{prop.taskName}</h2>
       </span>
       {typeof prop.taskPrompt === 'string' ? (
-        <p>{prop.taskPrompt}</p>
+        <p className="body task-prompt">{prop.taskPrompt}</p>
       ) : (
         prop.taskPrompt
       )}
       <div className="task-content" ref={ref}>
-        {fileSystem && (
+        {fileSystem && prop.displayFileSystem && (
```

Comment:
Nit: refactor this code you destructure arguments before you need to use them.
E.g. `const { displayFileSystem } = prop`
Feel free to do this in a follow-up PR 🙂